app/assets/javascripts/sufia/workflow_actions_affix.js

```diff
@@ -0,0 +1,12 @@
+Blacklight.onLoad(function() {
+  $(document).on('scroll', function() {
+    var workflowDiv = $('#workflow_controls');
+
+    var workflowDivPos = $('.workflow-actions').offset().top + $('.workflow-actions').height();
```

Comment:
we need a way to gracefully exit out of here if $('.workflow-actions') isn't found.